media/renderer.js

```diff
@@ -1,6 +1,16 @@
 const { ipcRenderer } = require('electron');
 
 const path = require('path');
+
+// Workaround: On some computers the audio system may go to sleep and
+// take a short while to activate. This can cut off the beginning of
+// clips, so we "play" a looping silent sound to prevent this.
+if (process.env.SCREENCRASH_NO_AUDIO !== 'true') {
```

Comment:
Can we add an environment variable to turn on (or off) this behaviour?

Reply:
Done. I opted to have the default be "on", since forgetting to turn it on when it's needed is worse than vice versa.